modules/manage/pages/disaster-recovery/shadowing/setup.adoc

```diff
@@ -105,17 +105,30 @@ endif::[]
 To set up Shadowing, you need to create a shadow link and configure filters to select which topics, consumer groups, and ACLs to replicate. 
 
 ifdef::env-cloud[]
-You can set up Shadowing with the Redpanda Cloud UI, `rpk`, or the Data Plane API. 
+You can set up Shadowing with the Redpanda Cloud UI, `rpk`, or the link:/api/doc/cloud-controlplane/topic/topic-cloud-api-overview[Redpanda Cloud API].
```

Comment:
This could be replaced with the partial `shadow-link-cloud-tip.adoc`, but would you please edit that partial to use this Redpanda Cloud API link (instead of Data Plane API)?